src/app/shared/header/header.component.html

```diff
@@ -19,7 +19,10 @@
 <header class="mdm-header">
   <div class="mdm-header__logo">
     <a [routerLink]="logoLink?.routerLink ?? ''">
-      <img [src]="logoLink?.imageSrc" [alt]="logoLink?.label" />
+      <img
```

Comment:
The custom logo functions well. I think there should be some size restrictions though to ensure the header is not growing too large. If someone is able to upload any  kind of image, they could upload one where the dimensions are unsuitable. For example:

![image](https://user-images.githubusercontent.com/3219480/226893934-0f119b07-bc73-4b41-96c2-c7051d83f1c0.png)

Perhaps add a CSS class that imposes `max-width` and/or `max-height` in the logo region, but open to other suggestions.

Reply:
My view is that an image should be correctly sized before it appears in the header. This is to be used for theming purposes, so I don't think an image should be being resized. But I guess this is open to abuse.

Reply:
Based on your opinion, I would at least suggest the following short-term edits:

1. Document what maximum image size is expected to at least have it recorded somewhere, then assume that the end user will adhere to that.
2. (Optional) Consider the layout of the rest of the header with different sizes of logos. In the above example, the links could be aligned vertical centre for instance. Happy for that to be a separate bug issue to resolve later though.

Reply:
Updated `docs/theming.md` to define maximum logo size.